docs/content/components/section.mdx

```diff
@@ -0,0 +1,131 @@
+
+usage: |
+  * The section component should be given a label to describe the content and or function served by the component, unless the components within adequately describe this information.
+  * Give your section a heading to provide a shortened title for the initiative or message.
+  * Give your section a description to provide a concise description of the initiative or message.
+
+  ### Where to use
+
+  * Use component on home, landing, and inner pages.
+
+  ### When to use
+
+  * Use when a section of a webpage doesn’t have a more semantically specific component available to describe it.
+  * Use to create an association between components, if there aren’t existing patterns that can be used to semantically describe the relation between specific components.
+  * Use to create a visual separation between different piece of content on a page.
+  * Use to give the "row" of your page vertical padding.
+
+  ### When to consider something else
+
+  * When a more semantically specific component is available for your content.
+    * Review our [list of NCIDS Components](https://designsystem.cancer.gov/components) for other options.
+  * When a component pattern exists that allows you to create an association between multiple components
+    * Review [NCIDS patterns](https://designsystem.cancer.gov/templates) for other options
+
+best_practices: |
+  Character limits are strongly recommended to ensure text is concise, scannable, and not visually distorted. Character limits are based on best practices and when a line of text will be forced to wrap onto a second line of text on desktop.
+  * Heading: 50 characters
+  * Description: 150 characters
+
+accessibility: |
+  * Use Aria-Labels to provide users with an accessible name for a section. This should match or refer to the heading of the section, most likely.
+  * Utilize the `<section>` element to ensure your webpage has appropriate structure.
+  * A `<section>` element should not be `<section>` element without an aria-label or aria-labelledby. If there is no label for the component, then a `<div>` element should be used instead.
+
+code_snippets:
+  elements:
+    - title: Default Section
+      intro: |
+        * The section component will default to be the full width of the component's container, without margins or horizontal padding.
+        * A component that covers the full width of the container without margins would likely be a direct child element of the section component.
+      preview: true
+      code: |
+        <div class="usa-section">
+          This is example text in a default section. Notice the lack of margins and the padding on the top and bottom of the section. Your HTML would go here in place of this text.
+        </div>
+    - title: Applying Standard Page Margins
```

Comment:
In testing with other developers, the `<section>` examples were missing. Please add 2 examples above the Applying Standard Page Margins example for:
* Making an Accessible Section using a Label
* Making an Accessible Section Labelled with Another Element

Feel free to use your discretion in adding good descriptions for each.